x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/transforms/SettingsConfig.java

```diff
@@ -98,9 +98,9 @@ public Integer getDatesAsEpochMillisForUpdate() {
 
     public ActionRequestValidationException validate(ActionRequestValidationException validationException) {
         // TODO: make this dependent on search.max_buckets
-        if (maxPageSearchSize != null && (maxPageSearchSize < 10 || maxPageSearchSize > 10_000)) {
+        if (maxPageSearchSize != null && (maxPageSearchSize < 10 || maxPageSearchSize > 65_536)) {
```

Comment:
The limit is `65_535`. You might stumbled upon https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket.html, which says `65_536`. I think this is wrong, I will check this.

Reply:
I turns out `65_536` is correct but the referenced page is wrong.

Reply:
To avoid future confusion, it would be nice to replace `65_536` with this constant: `MultiBucketConsumerService.DEFAULT_MAX_BUCKETS`

But this is nice to have, I can do this change myself as well.

Reply:
@hendrikmuhs, Thank you for the feedback! I took the `65_536` value from the issue #70706.

Reply:
Don't worry. I am making these changes and will commit soon